homeassistant/components/fordpass/__init__.py

```diff
@@ -0,0 +1,147 @@
+"""The FordPass integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+from dotted.collection import DottedDict
+from fordpass import Vehicle
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import DOMAIN, MANUFACTURER, VEHICLE, VIN
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+
+PLATFORMS = ["lock"]
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=300)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the FordPass component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up FordPass from a config entry."""
+    user = entry.data[CONF_USERNAME]
+    password = entry.data[CONF_PASSWORD]
+    vin = entry.data[VIN]
+
+    coordinator = FordPassDataUpdateCoordinator(hass, user, password, vin)
+
+    await coordinator.async_refresh()  # Get initial data
```

Comment:
You can add comments but this is so obvious.
```suggestion
    await coordinator.async_refresh()
```